src/client/clientlauncher.cpp

```diff
@@ -366,6 +366,20 @@ bool ClientLauncher::launch_game(std::string &error_message,
 	if (cmd_args.exists("password"))
 		menudata.password = cmd_args.get("password");
 
+
+	if (cmd_args.exists("password-file")) {
+		std::ifstream passfile(cmd_args.get("password-file"));
+		if (passfile.good()) {
+			passfile >> menudata.password;
```

Comment:
maybe trim the extra \n some can add a the end of the file with a file edit ?

Reply:
In my tests it never included any newlines, but also just found it will never include spaces either. Guess I'll find a new way to read a line from a file. Seems like getline() will work as it'll include spaces but no newlines.